examples/python/zsl_visual_prompting/README.md

```diff
@@ -0,0 +1,32 @@
+# Zero-shot Segment Anything example
+This example demonstrates how to use a Python API implementation of Segment Anything pipeline inference:
```

Comment:
```suggestion
This example demonstrates how to use a Python API implementation of Zero-shot Segment Anything pipeline inference:
```
I'm not sure it's better, so please ignore it if you think it isn't :)

Reply:
Added